### DISCUSSION TITLE
Convert projec to Laravel

### DISCUSSION THREAD

**Discussion by DubStepMad** [Donkie/Spoolman#403] [Ideas]
Hi, I'm willing to convert the Spoolman project over to Laravel including authentication so it could be exposed to the internet and have some security regarding this.

I'm fluent with Laravel from different projects I have done over the years but I've never converted another project before.

I was wondering if there is a best way to contact you @Donkie regarding the use of your Spoolman logo and other details if it was to go ahead with the conversion.

Thanks 😄

**Comment by Donkie** [comment]
This project is open source so you are free to fork it and do whatever you want.

That said, converting to Laravel sounds like a waste of time. Implementing authentication is entirely possible in the current code base and would not be less secure than any other method. I would not be interested in maintaining a PHP code base, so it would result in two forks of Spoolman being independently developed with differenting features, which does not sound so good for the community.

**Reply by DubStepMad** [reply]
Thanks for the reply.

I will take onboard what you mentioned and weigh up if the project is worth it as Python isn't my strongest language as I don't use it often.
